frb_example/pure_dart/flutter_rust_bridge.yaml

```diff
@@ -19,3 +19,5 @@ enable_lifetime: true
 dump_all: true
 # only for development of flutter_rust_bridge, not for normal users (use local dependency)
 local: true
+features:
+  - test
```

Comment:
nit: maybe we can rename this feature to something like `this_feature_is_for_test` or something else. because the current `test` may make readers confused - does it mean "enable this feature when we are running unit tests of this package"